glossary.md

```diff
@@ -94,6 +94,16 @@ instance: Imagine a satelite (`observing system`) that is very important to a se
 latency (`gaps`. Those would be linked by a thick (high `criticality`) red (low 
 `performance`) line. 
 
+
+```mermaid
+flowchart LR
+    A[Observing Systems]--> |supported by| B[Data Products]
+    A --> |self| A
+    B --> |supported by| C(Applications)
+    C --> C
+    C --> |supported by| D[Societal Benefit Areas]
```

Comment:
@hazelshapiro  you are able to edit here if you'd like.

Reply:
I agree with @hazelshapiro 's comments on the verb and arrow direction. If we use `supported by`, the arrow should go to the left, from the subject to the object. If we use `support` or `supports`, then the subject and object are reversed and the arrow should go right.

The latter seems more intuitive to me!

Reply:
agreed, implemented that on the latest commit.